test/Unit/Porter/Net/Http/HttpOptionsTest.php

```diff
@@ -65,6 +65,47 @@ public function testProxy()
         self::assertSame($host = 'https://foo.com:80', (new HttpOptions)->setProxy($host)->getProxy());
     }
 
+    public function testUserAgent()
+    {
+        self::assertSame($userAgent = 'Foo/Bar', (new HttpOptions)->setUserAgent($userAgent)->getUserAgent());
+    }
+
+    public function testFollowLocation()
+    {
+        $options = (new HttpOptions)->setFollowLocation(true);
+
+        self::assertTrue($options->getFollowLocation());
+        self::assertFalse($options->setFollowLocation(false)->getFollowLocation());
+    }
+
+    public function testRequestFullUri()
+    {
+        $options = (new HttpOptions)->setRequestFullUri(true);
+
+        self::assertTrue($options->getRequestFullUri());
+        self::assertFalse((new HttpOptions)->setRequestFullUri(false)->getRequestFullUri());
```

Comment:
Unlike in the previous test, the `$options` instance is not being reused for the `false` test. This is probably a mistake?